modules/lib/src/main/resources/assets/js/app/ContentStatusToolbar.ts

```diff
@@ -56,9 +67,19 @@ export class ContentStatusToolbar
     private updateStatus(content: ContentSummaryAndCompareStatus): void {
         this.status.setClass('status');
 
+        if (this.content) {
+            const clazz = this.content.getStatusClass();
+            this.showChanges.removeClass(clazz);
+            this.status.removeClass(clazz);
```

Comment:
`this.status.setClass()` has already set just one class on `this.status`, so there's no point in calling `this.status.removeClass` - it will never do anything.